### ISSUE TITLE
Colors in graph are too similar

### ISSUE THREAD

**Issue by MichaelMure** [buchen/portfolio#2707]
**Is your feature request related to a problem? Please describe.**

I find the time series hard to read because of the colors that are very similar. Here is an example where the generated palette is some variation of blue/violet/orange. Figuring out how which line match with the legend becomes a game of guessing which color match the most.

![image](https://user-images.githubusercontent.com/294669/155295609-bd19163d-fcf6-40fc-b030-cc9cb8717f79.png)

**Describe the solution you'd like**

A different algorithm to generate the palette would help a lot. There is some suggestion over here: https://softwareengineering.stackexchange.com/questions/198065/what-algorithms-are-there-for-picking-colors-for-plot-lines-on-graphs

It looks like the golden ratio algorithm would work quite well.

**Describe alternatives you've considered**

Another possibility (or rather, a complementary solution) would be to have some cue about which series is the closest to the mouse cursor in the tooltip window showing up when clicking on the graph.

![image](https://user-images.githubusercontent.com/294669/155296756-7425979a-3c8c-4cb6-a900-3054b72efa0a.png)

**Comment by funnym0nk3y** [comment]
You have to keep in mind that the colors could (and in my opinion should) also represent the taxonomy of the asset. 

![grafik](https://user-images.githubusercontent.com/41870754/156569716-94e6761a-7eed-47b8-903b-c76bc9dc5198.png)

**Comment by flywire** [comment]
> You have to keep in mind that the colors could (and in my opinion should) also represent the taxonomy of the asset.

I don't agree. Unless displayed colours can be distinguished from each other the chart is useless. It would be useful to click on legend item(s) and have the line change to bold.
